modules/video/src/optflowgf.cpp

```diff
@@ -433,13 +434,13 @@ FarnebackUpdateFlow_GaussianBlur( const Mat& _R0, const Mat& _R1,
     for( i = 0; i <= m; i++ )
         kernel[i] = (float)(kernel[i]*s);
 
-#if CV_SSE2
+#if CV_SIMD128
     float* simd_kernel = alignPtr(kernel + m+1, 16);
-    volatile bool useSIMD = checkHardwareSupport(CV_CPU_SSE);
+    volatile bool useSIMD = checkHardwareSupport(CV_CPU_SSE) || checkHardwareSupport(CV_CPU_VSX);
```

Comment:
These runtime checks are not needed anymore. `useSIMD = true` if this macro is defined.

Remove corresponding "if (useSIMD)" checks in this file (but lets leave code scope "{ ... }" to avoid large changes in the code and/or local variables conflicts)